src/common.cpp

```diff
@@ -1689,7 +1689,7 @@ bool is_forked_child(void) {
 
     bool is_child_of_fork = (getpid() != initial_pid);
     if (is_child_of_fork) {
-        printf("Uh-oh: %d\n", getpid());
+        wprintf(L"Uh-oh: %d\n", getpid());
```

Comment:
FWIW, I'd prefer that we never use the variants that use stdout implicitly. We should always use the "f" forms; e.g., `fwprintf()`. This makes it easier to just `grep stdout` or `grep stderr` without having to also worry about the places where stdout is implicit.